### PR TITLE
♻️ Maintenance/remove trafaret from repo

### PR DESCRIPTION
**Pull request by pcrespov**
## What do these changes do?

We decided to replace ``trafaret`` config library by ``pydantic.settings``. ``trafaret`` is basically introduced in many services via two packages, namely ``serviceslib[aiohttp]`` and ``simcore-sdk``. 

#### ♻️ Changes in ``simcore-sdk``
 - why it requires ``trafaret``? 
    - Because of ``simcore_sdk.config.db`` and ``simcore_sdk.configconfig.s3``. 
    - Those removed since ``settings-library`` counterparts exist
 - removes ``trafaret`` dependency in library and in several services downstream
   - ``api-server``, ``director-v2``, ``dynamic-sidecar``, ``tests/deploy-swarm`` affected 
 - ``packages/simcore-sdk version: 1.0.0 → 1.1.0``

#### ♻️ Changes in ``serviceslib[aiohttp]``
 - why it requires ``trafaret``? 
   - because of ``serviceslib.aiohttp.config_schema_utils``
   - only used in web-server so this module's functionality is moved to web-server
 - removes ``trafaret`` dependency in library and in donwstream to ``storage``  service
 -  ``servicelib version: 1.1.0 → 1.2.0'``

#### ``director`` and ``web/server``
- After this PR, only ``web/server`` service requires ``traferet`` and this will be removed in a separate PR for the sake of clarity
- ``director`` remains unchanged because it is deprecated
- 💚 ``web/server`` was installing ``simcore-sdk`` and not using it.
   -  this suggests that we do not have guarantees that the ``requirements`` is the exact installation !!!

## Related issue/s

- implementing some items of #2008

## How to test

standard tests cover these changes

## Checklist

- [x] Openapi changes? ``make openapi-specs``, ``git commit ...`` and then ``make version-*``)
- [x] Database migration script? ``cd packages/postgres-database``, ``make setup-commit``, ``sc-pg review -m "my changes"``
- [x] Unit tests for the changes exist
- [x] Runs in the swarm
- [x] Documentation reflects the changes
- [x] New module? Add your github username to [.github/CODEOWNERS](.github/CODEOWNERS)

### CODE REVIEW

**Review by sanderegg — APPROVED** [review]
good step!!

File: services/storage/requirements/_base.in [hunk 1]
```diff
@@ -4,14 +4,15 @@
 
 --constraint ../../../requirements/constraints.txt
 
+
 --requirement ../../../packages/models-library/requirements/_base.in
 --requirement ../../../packages/postgres-database/requirements/_base.in
+--requirement ../../../packages/settings-library/requirements/_base.in
 
 # settings-library[aiohttp]
```
Comment (GitHK) [inline services/storage/requirements/_base.in:12]: can you change the to service-library I think the comment is wrong

**Review by GitHK — APPROVED** [review]
1. If I get this, the webserver will directly install trafaret and will no longer come via  simcore-service-library.
2.director-v0 will be unchanged and use it from simcore-service-library.

**Comment by pcrespov** [comment]
> Looking nice. Just to be sure, see below questions:
> 
> 1. the webserver will directly install trafaret and will no longer come via  simcore-service-library.
> 2. director-v0 will be unchanged and use it from simcore-service-library.

1. yes. Notice that this dependency will disappear in the next PR
2. yes. Notice that simcore-service-library is pinned to an older version

**Pull request merged**